supervisionorders.md

```diff
@@ -33,7 +33,7 @@ Exposed links:
 * group: the group that can supervise the item
 
 Status codes:
-* 200 OK - if the item is found and it is visible to the current user or the anonymous user. Withdrawn items are returned
+* 200 OK - if the item is found and it is visible to the current user. Withdrawn items are returned
```

Comment:
The phrase "The withdrawn items are returned" must be removed. There is no chance that a supervision order is associated with a withdrawn item